router/core/request_tracing.go

```diff
@@ -23,11 +23,11 @@ func ParseRequestTraceOptions(r *http.Request) (options resolve.RequestTraceOpti
 	var (
 		values []string
 	)
-	if r.Header.Get(RequestTraceHeader) != "" {
+	if r.Header.Get(RequestTraceHeader) == "true" {
 		options.Enable = true
 		values = r.Header.Values(RequestTraceHeader)
 	}
-	if r.URL.Query().Get(RequestTraceQueryParameter) != "" {
+	if r.URL.Query().Get(RequestTraceQueryParameter) == "true" {
```

Comment:
I think this is not correct, any non-empty string is ok, except "false".

Reply:
I reverted it. I thought supporting false and true as values would be more reasonable because we never communicated that `wg_trace` is the same as `wg_trace=true`

Reply:
Ok